packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -103,6 +119,12 @@ export class BaseButton extends BaseComponent<IButtonProps, {}> implements IButt
   }
 
   protected onRenderContent(tag: any, buttonProps: IButtonProps): JSX.Element {
+    let {
```

Comment:
Nevermind, I see a bunch of 'const' use up at the render method. I'll make everything const that isn't getting reassigned.